components/inputs/test/input-number.test.js

```diff
@@ -739,8 +718,7 @@ describe('d2l-input-number', () => {
 		});
 
 		it('should handle different locales', async() => {
-			documentLocaleSettings.language = 'fr';
```

Comment:
@bearfriend @dlockhart Changes to the `testing` library made these tests start failing. I've updated them to set the language through `fixture` instead (and removed the extra waiting that shouldn't be needed anymore). But let me know if this was testing this method of setting the language specifically, and there's actually an issue in `testing`.

Reply:
Nope this is the perfect change!